front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,356 @@
+import React from 'react';
+
+import { Table, TR, TH, TD } from '@ag-media/react-pdf-table';
+import { Page, Text, Image, Document, View } from '@react-pdf/renderer';
+import { useTranslation } from 'react-i18next';
+
+import iconAlert from 'assets/simulationSheet/icon_alert_fill.png';
+import logoSNCF from 'assets/simulationSheet/logo_sncf_reseau.png';
+import type { PostStdcmApiResponse, RollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+import { formatDay } from 'utils/date';
+import { getStopDurationTime, getStopTime } from 'utils/timeManipulation';
+
+import styles from './SimulationReportStyleSheet';
+import { formatCreationDate } from '../utils';
+
+type SimulationReportSheetProps = {
+  stdcmData: PostStdcmApiResponse;
+  rollingStockData: RollingStockWithLiveries;
+  number: string;
+  mapCanvas?: string;
+};
+
+const SimulationReportSheet = ({
+  stdcmData,
+  rollingStockData,
+  number,
+  mapCanvas,
+}: SimulationReportSheetProps) => {
+  const { t } = useTranslation('stdcm-simulation-sheet');
+  const formattedParts = formatCreationDate(stdcmData.path.created);
+  const formattedDate = t('formattedDate', {
+    day: formattedParts.formattedDay,
+    month: formattedParts.formattedMonth,
+    year: formattedParts.year,
+    hours: formattedParts.formattedHours,
+    minutes: formattedParts.formattedMinutes,
+  });
+
+  let renderedIndex = 0;
+
+  const fakeInformation = {
+    rcName: 'Super Fret',
+    rcPersonName: 'Jane Smith',
+    rcPhoneNumber: '01 23 45 67 89',
+    rcMail: 'john.doe@example.com',
+    path_number1: 'n°123456',
+    path_number2: 'n°987654',
+  };
```

Comment:
You can move this outside of SimulationReportSheet :)

Reply:
fixed